### PR TITLE
feat(oai): convert collect_images to asset

### PR DESCRIPTION
**Pull request by PaulHax**
image

@dzenanz

### CODE REVIEW

File: knee_sarg/assets/oai.py [hunk 1]
```diff
@@ -287,3 +290,52 @@ def has_image_files(
     ),
     ins={"runs": AssetIn("cartilage_thickness_runs")},
 )
+
+
+class CollectImagesConfig(Config):
+    out_dir: str = Field(
+        default_factory=lambda: "all-images",
+        description="JSON file with array of patient IDs",
```
Comment (dzenanz) [inline knee_sarg/assets/oai.py]: dir not file

File: knee_sarg/assets/oai.py [hunk 2]
```diff
@@ -287,3 +290,52 @@ def has_image_files(
...
+    )
+    files_to_collect: List[str] = Field(
+        default_factory=lambda: THICKNESS_IMAGES,
+        description="List of files to copy into out directory",
```
Comment (dzenanz) [inline knee_sarg/assets/oai.py]: Better description: `Name of files that appear in each case's out directory that are to be copied into "all-images".`.

File: knee_sarg/assets/oai.py (hunk 1 above)
Comment (PaulHax) [reply knee_sarg/assets/oai.py]: 🙏

File: knee_sarg/assets/oai.py (hunk 2 above)
Comment (PaulHax) [reply knee_sarg/assets/oai.py]: yes, better, changed.

**Comment by dzenanz** [comment]
Looks good. I can neither "resolve" my comments, nor "approve" the PR.

**Pull request merged**